stdlib/contrib/mhall119/helpers/helpers.flux

```diff
@@ -0,0 +1,94 @@
+// Package helpers provides shortcut functions for small common tasks
+//
+// ## Metadata
+// introduced: ??
+package helpers
+
+import "array"
+import "json"
+// yieldValue lets you display a scalar in the InfluxDB UI by
+// wrapping it in an Array
+//
+// ## Parameters
+// - v: Input data.
+// - name: Name to pass to the subsequent yield() call.  Default is 'value'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldValue(v: testData.testValue)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldValName="value"
+yieldValue = (v, name=defaultYieldValName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: v},
+      ]
+    )
+    |> yield(name: name)
+
+// yieldObject lets you display an object in the InfluxDB UI by
+// encoding it as String
+//
+// ## Parameters
+// - o: Object to yield
+// - name: Name to pass to the subsequent yield() call.  Default is 'object'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldObject(v: testData)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldObjName="object"
+yieldObject = (o, name=defaultYieldObjName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: display(v: o)},
+      ]
+    )
+    |> yield(name: name)
+
+// yieldObject lets you display an object in the InfluxDB UI by
+// encoding it as a JSON string and wrapping it in an Array
+//
+// ## Parameters
+// - o: Object to yield
+// - name: Name to pass to the subsequent yield() call.  Default is 'object'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldObject(v: testData)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldJSONName="json"
+yieldJSON = (o, name=defaultYieldJSONName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: string(v: json.encode(v: o))},
+      ]
+    )
+    |> yield(name: name)
```

Comment:
As for how to test these with the explicit `yields`. How important is it that these have calls to yield? Seems the real value the function provides is the boilerplate of getting the value displayed in a table. Could the yield be implicit or explicit on the caller side? That generally fits better with the Flux idiomatic code as it makes it easier to compose these functions with other functions.

Reply:
I don't think there's many scenarios where a developer won't want to yield this data immediately, and since the wrapper functions are already very small the yield itself is like 1/3 of what they do.

I think these helper functions are more valuable if they include the yield themselves.

Reply:
@jsternberg Is there a way we can trick the test engine into testing these functions with yields in them?

Reply:
I don't think so.